gax-httpjson/build.gradle

```diff
@@ -3,10 +3,12 @@ archivesBaseName = "gax-httpjson"
 
 dependencies {
   compile project(':gax'),
+    libraries.gson,
     libraries.guava,
     libraries.jsr305,
     libraries.autovalue,
     libraries.threetenbp,
+    libraries.apiClient,
```

Comment:
What is this dependency necessary for?

Reply:
It was for GoogleNetHttpTransport, but a simpler NetHttpTransport seems to work fine and accept credentials, so I've removed GoogleNetHttpTransport and the dependency.